config/config.go

```diff
@@ -1375,6 +1375,15 @@ func GetSafeClusterName(cluster string) string {
 	return cluster
 }
 
+// IsValidationsEnabled checks the value of ValidationReconcileInternal
+// Validations are enabled for values higher than 0, otherwise validations will be disabled
+func (conf Config) IsValidationsEnabled() bool {
+	if conf.ExternalServices.Istio.ValidationReconcileInterval != nil && *conf.ExternalServices.Istio.ValidationReconcileInterval > 0 {
+		return true
+	}
+	return false
```

Comment:
nit: more concise but not necessary
```suggestion
        return conf.ExternalServices.Istio.ValidationReconcileInterval != nil && *conf.ExternalServices.Istio.ValidationReconcileInterval > 0
```